portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <div class="topnav">
+        <a class="active" href="index.html">Home</a>
+        <a href="second.html">About me</a>
+        <a href="projects.html">Projects</a>
+        <a href="skills.html">Skills</a>
+        <a href="#contact">Contact</a>
+    </div>
+</head>
+<header id="contact">
+    <h1> Contact Info </h1><br>
+    <p> Email: Abdi0122@umn.edu
+        <p>
```

Comment:
This is a little strange. I think you may have accidentally using opening brackets instead of closing ones. Please fix.